test/test_spectral_ops.py

```diff
@@ -340,6 +348,164 @@ def test_fn(x):
 
                 self.assertTrue(torch.autograd.gradcheck(test_fn, (input,)))
 
+    # nd-fft tests
+
+    @skipCPUIfNoMkl
+    @skipCUDAIfRocm
+    @unittest.skipIf(not TEST_NUMPY, 'NumPy not found')
```

Comment:
You may need `@onlyOnCPUAndCUDA` here and on another test below. Would you rebase to test the XLA build?

Reply:
Noticed some of the `fft` tests were missing this as well. Added and rebased on `viable/strict`. 

@mruberry

Reply:
Thanks. XLA build looks good.